stdlib/2/UserString.pyi

```diff
@@ -1,4 +1,70 @@
 import collections
+from typing import Any, Iterable, List, MutableSequence, Sequence, Optional, overload, Text, TypeVar, Tuple, Union
 
-class UserString(collections.Sequence): ...
-class MutableString(UserString, collections.MutableSequence): ...
+_UST = TypeVar("_UST", bound=UserString)
+_MST = TypeVar("_MST", bound=MutableString)
+
+class UserString(Sequence[UserString]):
+    def __init__(self, seq: object) -> None: ...
+    def __int__(self) -> int: ...
+    def __long__(self) -> long: ...
+    def __float__(self) -> float: ...
+    def __complex__(self) -> complex: ...
+    def __hash__(self) -> int: ...
+    def __len__(self) -> int: ...
+    @overload
+    def __getitem__(self: _UST, i: int) -> _UST: ...
+    @overload
+    def __getitem__(self: _UST, s: slice) -> _UST: ...
+    def __add__(self: _UST, other: object) -> _UST: ...
+    def __radd__(self: _UST, other: object) -> _UST: ...
```

Comment:
I would make these two `Any` instead of `object` to simplify subclassing.

Reply:
All done (same for the other comments below). Thanks for reviewing!